spec/footer_spec.rb

```diff
@@ -8,6 +10,8 @@
     @header.click_my_account
     @logon.login('akratz@mcfadyen.com', 'Admin123') # info provisória
     visit '/'
+    @footer_met = FooterMethods.new
+    @footer_as = FooterAssertions.new
   end
 
   after(:each) do
```

Comment:
add visit method here in order to execute before each test block and to ensure that the footer is visible before perform click action.